frontend/views/pages/GroupSettings.vue

```diff
@@ -58,10 +58,8 @@ page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='currentGroupS
         data-test='saveBtn'
       ) Save changes
 
-  page-section(title='Leave Group')
-    p
-      | {{ L('This means you will stop having access to the') }} #[b {{ L('group chat') }}] {{ L('including direct messages to other group members) and') }} #[b {{ L('contributions') }}].
-      | {{ L('Re-joining the group is possible, but requires other members to vote and reach an agreement.') }}
+  page-section(:title='L("Leave Group")')
+    p(v-html="L('This means you will stop having access to the <b>group chat</b> (including direct messages to other group members) and <b>contributions</b>. Re-joining the group is possible, but requires other members to vote and reach an agreement.')")
```

Comment:
NOTE: I have changed this (since the above doesn't work very well in my experience, especially not with variables as well), in my PR #611 — now the way to this is simple:

```pug
i18n(tag='p' html='This means you will stop having access to the <b>group chat</b> (including direct messages to other group members) and <b>contributions</b>. Re-joining the group is possible, but requires other members to vote and reach an agreement.')
```

I've also fixed this in my PR.